pkg/security/apis/security/types.go

```diff
@@ -38,6 +38,10 @@ type SecurityContextConstraints struct {
 	// You must not list a capability in both AllowedCapabilities and RequiredDropCapabilities.
 	// To allow all capabilities you may use '*'.
 	AllowedCapabilities []kapi.Capability
+	// AllowedFlexDrivers is a whitelist of allowed Flexvolume drivers.
+	// Empty or nil indicates that all drivers may be used.
+	// +optional
+	AllowedFlexDrivers []string
```

Comment:
making this an array of structs lets us expand options in the future (like setting the readOnly option), similar to the discussion around allowedHostPaths.